### PR TITLE
smoke testing about services

### PR DESCRIPTION
**Pull request by lufercc**

### CODE REVIEW

File: test/smoke/servicesSt.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    };
+
+var serviceId;
+var token;
+var serviceType;
+
+
+
+describe('Room Manager Smoke Test:', function() {
+    this.timeout(50000);
```
Comment (JFVF) [inline test/smoke/servicesSt.js]: I would suggest to have this in a config file

File: test/smoke/servicesSt.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+                expect(status).to.equal(200);
+                done();
+            });
+    });
+
+    /**
+     * verify that the API services by Id exist
+     * */
+
+    it('verify that the API services by Id exist', function(done){
```
Comment (JFVF) [inline test/smoke/servicesSt.js]: I would suggest to change the name of the test
GET API Services exit for instance

File: test/smoke/servicesSt.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+                });
+        });
+
+        /**
+         *this describe test the existence of the API when did post
+         */
+
+        it('verify that the API to be post exist', function(done){
+            servicesLib
+                .postservices(token,serviceType,loginExchange)
```
Comment (JFVF) [inline test/smoke/servicesSt.js]: are you adding the service twice?

File: test/smoke/servicesSt.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+        "hostname": settings.domain+'.lab'
+    };
+
+var serviceId;
+var token;
+var serviceType;
+
+
+
+describe('Room Manager Smoke Test:', function() {
```
Comment (JFVF) [inline test/smoke/servicesSt.js]: Please review the order of the test, basically the logic.

**Comment by JFVF** [comment]
Accepting but take a look on the comments and make the changes

File: test/smoke/servicesSt.js (hunk 3 above)
Comment (lufercc) [inline test/smoke/servicesSt.js]: on before add one because in one test delete it 
and on after delete other because there is a test that create it

**Pull request merged**